### ISSUE TITLE
Do taxonomy fields need to be published through the Content Type Hub to be leveraged within auto-apply policy KQL conditions?

### ISSUE THREAD

**Issue by kstat** [MicrosoftDocs/microsoft-365-docs#10234]
Mentioned in bullet #6 in reference here: [Important things to know](https://learn.microsoft.com/en-us/microsoft-365/community/auto-apply-retention-labels-in-office-365-using-content-types-and-metadata#important-things-to-know)

Do Taxonomy site columns need to be published through the Content Type Hub (CTH) in SPO to be able to map their resulting Search Crawled properties to Search Managed Properties for purposes of defining KQL conditions in an Auto-apply policy?

**Comment by yogkumgit** [comment]
@daisyfell Please help us in resolving this. Thanks

**Comment by daisyfell** [comment]
Per Marc Anderson:

The short answer is no.

Any Site Column you create - of any type - becomes a Crawled Property once there is content with values in that column. (The crawling may lag a bit, but it will be crawled.) The decision about whether to create a Site Column at the site level or the tenant level (CTH) should be based on how widely it will be used and where it should be managed. 
 
Something that many people don't realize is if you create Site Columns in multiple sites with the same InternalName, they are treated as equivalent by the search crawler. So, it may make sense in some cases to just create the Site Column in the small number of sites where it is needed. Site Columns aren't published from the CTH, whereas Content Types are.
 
You can find more details on this here: https://learn.microsoft.com/en-us/microsoft-365/community/how-do-site-columns-become-managed-properties-thus-available-for-search

**Comment by sympmarc** [comment]
If there are additional questions here about the policy side of the question, @joannecklein may be able to help.

**Comment by joannecklein** [comment]
I tested applying a retention label via a KQL condition with different column data types a few years ago and at the time the limitation I found with a managed metadata column was although the column itself didn't need to be published thru a content type from the tenant-level content type gallery, the term set the column referred to needed to be defined at the tenant-level (as opposed to the site collection level). I don't believe this has changed. This could be because the retention label auto-apply is a tenant-level construct although I don't know for sure.
